src/error.rs

```diff
@@ -37,6 +39,10 @@ pub enum Error {
 
     /// Partition is deleted
     PartitionDeleted,
+
+    #[cfg(feature = "ssi_tx")]
+    /// SSI transaction-related error
+    Ssi(SsiError),
```

Comment:
The oracle error is just { Poisoned }.

How can the oracle realistically be poisoned? Maybe it can just use crate::Error::Poisoned?

Or maybe we can just .expect() the write_serializez_lock to never poison...

Reply:
I switched it to `crate::Error::Poisoned`. Even if, in theory, it can't ever happen, I still prefer to have it covered.